src/templates/ratings/add.html

```diff
@@ -26,7 +26,7 @@
       </p>
       <p class="rating-label">{{ _('Please select a star rating.') }}</p>
       <textarea id="review-body" rows="2" cols="40" name="body"
-                placeholder="{{ _('Tell us what you love about this app!') }}"
+                placeholder="{{ _('Tell us What you like about this app. Write like you’re telling a friend about your experience with the app. Give specific details, for example, What features did you like or dislike?') }}"
```

Comment:
Can you please remove the capitalisation of the word "what". There are two places it is capitalised.